src/test/rpc/Handler_test.cpp

```diff
@@ -0,0 +1,133 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2023 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <ripple/beast/unit_test.h>
+#include <ripple/rpc/impl/Handler.h>
+#include <test/jtx.h>
+
+#include <chrono>
+#include <iostream>
+#include <limits>
+#include <numeric>
+#include <random>
+
+namespace ripple::RPC {
+
+// NOTE: there should be no need for this function;
+// `std::cout << some_duration` should just work if built with a compliant
+// C++20 compiler. Sadly, we are not using one, as of today
+// TODO: remove this operator<< overload when we bump compiler version
+std::ostream&
+operator<<(std::ostream& os, std::chrono::nanoseconds ns)
+{
+    return (os << ns.count() << "ns");
+}
+
+// NOTE This is a rather naiive effort at a microbenchmark. Ideally we want
+// Google Benchmark, or something similar. Also, this actually does not belong
+// to unit tests, as it makes little sense to run it in conditions very
+// dissimilar to how rippled will normally work.
+// TODO as https://github.com/XRPLF/rippled/issues/4765
+
+class Handler_test : public beast::unit_test::suite
```

Comment:
I don't think the test class shouldn't be in the `ripple::RPC` namespace. `ripple::test` makes more sense.

Reply:
Thanks, moved to `test`